extension/src/popup/components/manageAssets/SearchAsset/index.tsx

```diff
@@ -58,32 +61,36 @@ const ResultsHeader = () => {
 
 export const SearchAsset = () => {
```

Comment:
I noticed that if you refresh this page and then try to search, it throws an error due to a missing asset list